### DISCUSSION TITLE
Existing Private GitHub Repository doesn't have wiki tab

### DISCUSSION THREAD

**Discussion by Namine7** [community/community#73321] [Repositories] [answered]
### Hi everyone,

When I created a private repository, I didn't have the education plan. So my Repository did not have a wiki tab. Now I upgraded my plan to an education plan and I can't or don't know how to add the wiki tab for that repository.. or do I have to create a new repository and transfer all data in order to get the wiki tab? 

Thank you for your help!
Best regards,
Patricia

**Comment by airtower-luna** [comment, marked as answer]
Check the settings of your repository. If you scroll down to the "Features" section there should be checkboxes for wiki and a bunch of other things.
